server/src/main/resources/reference.conf

```diff
@@ -1,4 +1,16 @@
 broccoli {
   nomad.url = "http://localhost:4646"
   templates.path = "./templates"
+  instances {
+    polling.frequency = 1
+    storage {
+      type = "fs"
+      fs.url = "./instances"
+      couchdb {
+        url = "http://localhost:5984"
+        dbName = "broccoli_instances"
+      }
+    }
+    parameters.defaultType = "string"
+  }
```

Comment:
- [ ] Now that we have these changes in the `reference.conf` I think it is time to either add the new parameter to the README or change the README to point to the `reference.conf` for configuration reference and remove the long tables I built. If we do that, we should at least put some comments though, so we don't lose any information that is currently provided.
- [ ] We should also add a section to the Wiki migration guide to 0.7.0, explaining the defaultType. 
- [ ] The new option in `meta.json` should also be explained in the Wiki, no?